integration-box/csv_import_via_http/http_csv_download_sample.dig

```diff
@@ -0,0 +1,15 @@
+timezone: "Asia/Tokyo"
+
+schedule:
+ daily>: 07:45:00
+
++task1:
+  py>: import.main
+  _env:
```

Comment:
These variables are better to be in arguments of main function except for apikey.